docs/source/_templates/txn_submit_tutorial.rst

```diff
@@ -73,27 +38,17 @@ we can generate the actual bytes to use for a key.
    private key is kept secret and secure.
 
 In addition to a private key, you will need a shareable public key generated
-from the private key. It should be encoded as a hexadecimal string, to
-distribute with the Transaction and confirm that its signature is valid.
-
-{% if language == 'JavaScript' %}
-
-.. code-block:: javascript
-
-    const publicKeyBytes = secp256k1.publicKeyCreate(privateKeyBytes)
-
-    const publicKeyHex = publicKeyBytes.toString('hex')
-
-{% else %}
+from the private key, which can be used to confirm a signature is valid, and
+signed by the person who said they did. Typically, this can be generated by
```

Comment:
Minor quibble: person is singular, they plural. Usually an attempt to avoid gender. Suggestion: "and signed by the person who claimed to have signed it." ;)